cmake/modules/autopas_lcov.cmake

```diff
@@ -0,0 +1,20 @@
+#if(AUTOPAS_ENABLE_COVERAGE)
+#    include(autopas_lcov)
+#endif()
```

Comment:
```suggestion
if(NOT AUTOPAS_ENABLE_COVERAGE)
    return
endif()
```

Reply:
Ah I see 👍 Done in [5c31535](https://github.com/AutoPas/AutoPas/pull/888/commits/5c31535601f4ed3b03c0ce8da116edf56ca44ee7)